### PR TITLE
[docs] Documentation on configuring functions runtime

### LINKED ISSUE #3073: [Question] cluster mode fuctions in Kubernetes cluster
Hi, 

I can't see any documents introduce running pulsar function in cluster mode along side with broker in Kubernetes. But from broker configmap it looks like function is supported. How to use this? Do I need to customize broker image to add my jar file and yaml or something else? Thanks a lot.

image

### LINKED ISSUE #4171: Pulsar Functions documentation does not mention deploying Functions workers in containers
**Describe the bug**
The current Pulsar Functions deployment documentation (https://pulsar.apache.org/docs/latest/functions/deployment/) lists only localrun and cluster mode as the deployment options for Functions. Should also note that Functions can be deployed to run in containers/nodes that are separate from the broker nodes.

**Expected behavior**
Functions deployment documentation needs to indicate that Functions can be deployed to run in containers/nodes that are separate from the broker nodes.

### PR DESCRIPTION
**Pull request by sijie**
Fixes #4171
Fixes #3073

### CODE REVIEW

**Comment by sijie** [comment]
@Jennifer88huang ^

File: site2/docs/functions-runtime.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+  pythonInstanceLocation:
+  # change the extra dependencies location:
+  extraFunctionDependenciesDir:
+```
+
+Currently Pulsar supports running Java, Python, and Go functions in Process runtime mode.
+
+## Configure Kubernetes runtime
+
+Configuring Kubernetes runtime is as simple as configuring the other two runtimes. You can just uncomment
```
Comment (Jennifer88huang) [inline site2/docs/functions-runtime.md]: What's the meaning of "uncomment"?

Comment (jiazhai) [reply site2/docs/functions-runtime.md]: :) try google it. @Jennifer88huang 
```
https://www.google.com.hk/search?newwindow=1&safe=strict&ei=-9PYXJWHEPKwmAWSqZdQ&q=uncomment&oq=uncomment&gs_l=psy-ab.3..0i10i70i255j0i10l3j0i67j0l5.1725.1725..2054...0.0..0.104.104.0j1......0....1..gws-wiz.t1uIqcWY1uA
```

Comment (Jennifer88huang) [reply site2/docs/functions-runtime.md]: @jiazhai Thank you very much for your information. 
When you ''uncomment" sth in code, you actually want to enable some configuration options; in some cases, you just disable it with "comment", that's why we use comment lines in code. Here I want to tell @sijie, he could use "enable" / "disable" or other similar words to replace "uncomment"/"comment" in technical writing. When he explains the meaning, he will realize it.

Comment (sijie) [reply site2/docs/functions-runtime.md]: @Jennifer88huang I think from technical writing perspective, "enabling" and "disabling" are probably better. However sometimes "enabling" and "disabling" are too general that users have no idea to enable or disable. Can we use a phrase like "enabling sth by uncommenting ..."?

**Comment by merlimat** [comment]
run integration tests
run java8 tests

File: site2/docs/functions-runtime.md (hunk 1 above)
Comment (Jennifer88huang) [reply site2/docs/functions-runtime.md]: Sure, it's much better.

**Comment by merlimat** [comment]
run java8 tests

**Comment by sijie** [comment]
run java8 tests

**Comment by sijie** [comment]
run java8 tests

**Comment by sijie** [comment]
run java8 tests

**Comment by sijie** [comment]
run integration tests

**Pull request merged**